sdk/tables/azure-data-tables/tests/test_table_client.py

```diff
@@ -210,9 +209,105 @@ def test_error_handling(self, tables_storage_account_name, tables_primary_storag
             with pytest.raises(ClientAuthenticationError):
                 service_client.create_table_if_not_exists(table_name="TestInsert")
 
+    def check_request_auth(self, pipeline_request):
+        assert f"/?{self.sas_token}" not in pipeline_request.http_request.url
```

Comment:
I would just use:
```suggestion
        assert self.sas_token not in pipeline_request.http_request.url
```
Just in case there are any other query parameters in the URL.